### PR TITLE
Updated Access Modal to not go back to the access list

### LINKED ISSUE #995: Edit Access Routing.
To edit access, you need to click on details.
When you click "Edit Access", the edit dialog appears, but behind that the page turns to the access list.

It should remain and update the details page

### PR DESCRIPTION
**Pull request by wongband**
closes #995

### CODE REVIEW

File: Keas.Mvc/ClientApp/components/Access/AccessDetails.tsx [hunk 1]
```diff
@@ -1,129 +1,151 @@
...
   updateSelectedAccess: (access: IAccess, id?: number) => void;
 }
 
-export default class AccessDetails extends React.Component<IProps, {}> {
-  public static contextType = Context;
-  public context!: React.ContextType<typeof Context>;
+const AccessDetails = (props: IProps) => {
+  const context = useContext(Context);
+  const history = useHistory();
+  const [shouldOpenEditModal, setEditModal] = useState<boolean>(false);
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Access/AccessDetails.tsx:26]: as a convention the useState should be `variableName` and `setVariableName` so it forms an easy to find pair. In this example that would mean naming them `const [shouldOpenEditModal, setShouldOpenEditModal]`

**Review by srkirkland — APPROVED** [review]
Otherwise maybe wait for Jason to take a look functionally tomorrow but the rest looks good to me and I like the move to hooks too

**Review by jSylvestre — CHANGES_REQUESTED** [review]
I'm getting an error assigning access.
I'll try to debug into it....

**Comment by wongband** [comment]
@jSylvestre what bug are you getting?

**Comment by jSylvestre** [comment]
> @jSylvestre what bug are you getting?

@wongband It is happening here:
https://github.com/ucdavis/Peaks/blob/master/Keas.Mvc/Controllers/Api/AccessController.cs#L159

**Review by jSylvestre — APPROVED** [review]
I'll merge once all the checks finish

**Pull request merged**